### ISSUE TITLE
Missing context typing at plugin level

### ISSUE THREAD

**Issue by alvar-osz** [graphql-hive/envelop#419]
**Describe the bug**

Following the documentation from `Typescript Support` I'm using this code example but the `onExecute` function has a context value of any, while `onParse` is correctly typed.

**To Reproduce**

```typescript
export const useMyPlugin = (): Plugin<{ myContext: string }> => {
  return {
    onParse({ context }) {
      // Here, context is typed as { myContext: string }
    },
    onExecute({ args }) {
      // Here, args.contextValue is NOT typed as { myContext: string }
    }
  };
};
```

**Expected behavior**

```typescript
export const useMyPlugin = (): Plugin<{ myContext: string }> => {
  return {
    onParse({ context }) {
      // Here, context is typed as { myContext: string }
    },
    onExecute({ args }) {
      // Here, args.contextValue SHOULD be typed as { myContext: string }
    }
  };
};
```

**Environment:**

- OS: macOS 11.4
- `@envelop/core`: 0.4.0
- NodeJS: v16.4.0

**Comment by dotansimha** [comment]
Hi @ajotaos ! Sorry for the inconvenience! I used the original `ExecutionArgs` and it has `contextValue?: any` by default.

I fixed that in: https://github.com/dotansimha/envelop/pull/420

**Comment by dotansimha** [comment]
Fixed in `@envelop/types@0.4.1` (and `@envelop/core@0.5.1`). 

@ajotaos let me know if that works for you :) I tested on the internal plugins and it seems to work!
